pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -254,7 +254,7 @@ func (e *EndpointController) deletePod(obj interface{}) {
 	}
 	pod, ok := tombstone.Obj.(*v1.Pod)
 	if !ok {
-		utilruntime.HandleError(fmt.Errorf("Tombstone contained object that is not a Pod: %#v", obj))
+		utilruntime.HandleError(fmt.Errorf("Tombstone contained object that is not a Pod: %#v", tombstone))
```

Comment:
It is widely existed in the controllers. So if necessary, I would like to expand this PR to contain them all. The modification may show a clean and accurate error print.